### DISCUSSION TITLE
Display day view similar to MS Outlook / Google etc.

### DISCUSSION THREAD

**Discussion by mzurhorst** [totaldebug/atomic-calendar-revive#908] [Feature Requests]
Hi.  

I am looking for a "day view" which looks similar to the view in MS Outlook / Google etc, where I would have a timeline on the left and have larger or smaller blocks depending on the duration of an event.

Can this be configured with this calendar card?

**Comment by marksie1988** [comment]
No that is not possible with the card, you can only display events that exist in that day

**Reply by mzurhorst** [reply]
Not sure how this is related to "events that exist in that day" since I am only talking about 1 day.  
I mean something like this: 
![grafik](https://user-images.githubusercontent.com/575190/212542386-9365ecd4-c03d-478b-906a-a175d2b3c943.png)

**Reply by marksie1988** [reply]
You can't do that, only displays the events for the day, not a timeline

**Reply by mzurhorst** [reply]
Ok, thank you.
